core/src/main/java/software/amazon/awssdk/retry/SdkDefaultRetrySettings.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2010-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.retry;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import software.amazon.awssdk.RetryableException;
+import software.amazon.awssdk.http.HttpStatusCodes;
+
+// TODO: Add some JDK9 sweetness to this class when ready
+public class SdkDefaultRetrySettings {
+
+    /**
+     * When throttled retries are enabled, each retry attempt will consume this much capacity.
+     * Successful retry attempts will release this capacity back to the pool while failed retries
+     * will not.  Successful initial (non-retry) requests will always release 1 capacity unit to the
+     * pool.
+     */
+    public static final int THROTTLED_RETRY_COST = 5;
+
+    /**
+     * When throttled retries are enabled, this is the total number of subsequent failed retries
```

Comment:
This comment seems misleading. If each retry consumes 5 cost then there would be only 20 retries before capacity is drained right?

Reply:
Capacity is THROTTLED_RETRIES * THROTTLED_RETRY_COST

Reply:
Ahhh makes sense.